src/drive/web/modules/views/Folder/FolderViewBody.jsx

```diff
@@ -53,12 +79,16 @@ const FolderViewBody = ({
       const isNote = models.file.isNote(file)
       const isShortcut = models.file.isShortcut(file)
 
-      //Should only be called if mobile
       if (isShortcut) {
-        const resp = await client.query(
-          Q('io.cozy.files.shortcuts').getById(file.id)
-        )
-        window.location.href = resp.data.attributes.url
+        if (isMobileApp()) {
+          const resp = await client.query(
+            Q('io.cozy.files.shortcuts').getById(file.id)
+          )
+          window.location.href = resp.data.attributes.url
+        } else {
+          const url = generateFileUrl({ file, client, isFlatDomain: true })
+          window.open(url, '_blank')
```

Comment:
sometimes we use `system`, sometimes `self` and now `blank`. Hard to follow

Reply:
Do we use `self` at all? But yeah we do use `system` in one place, not sure if we could use `blank` instead there.

Reply:
Yep in QuotaAlert https://github.com/cozy/cozy-ui/blob/master/react/QuotaAlert/index.jsx#L34 / 
https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/upload/QuotaAlert.jsx#L34 but I can't remember why :/